GLOSSARY.md

```diff
@@ -0,0 +1,25 @@
+The SOCI project introduces several new terms that sometimes have subtle differences between them. This glossary defines these terms.
+
+## Terminology 
+
+__SOCI__: Seekable OCI (pronounced so-CHEE). SOCI combines an unmodified [OCI Image](https://github.com/opencontainers/image-spec/blob/v1.0.2/spec.md) (or Docker v2 image) with a SOCI index to enable the SOCI snapshotter to lazily pull the image at runtime.
+
+__SOCI Index__: The combined set of metadata that enables lazy loading of an unmodified OCI image. Conceptually, the SOCI Index is the mapping between files and their locations within a compressed image. In practice, a SOCI Index is made up of a SOCI Index Manifest and zero or more zTOCs. Within a SOCI Index there may be zero or one zTOCs per image layer.
+
+__SOCI Index Manifest__: An [ORAS manfest](https://github.com/oras-project/artifacts-spec/blob/v1.0.0-rc.2/artifact-manifest.md) (soon to be an [OCI Reference Types manifest](https://github.com/opencontainers/wg-reference-types/blob/256c257cc8b725fd324722ee40ead6925b1c8ad8/docs/proposals/PROPOSAL_E.md)) containing the list of zTOCs in the SOCI Index as well as a reference to the image for which the manifest was generated. 
+
+__zTOC__: A Table of Contents for compressed data. A zTOC contains a set of spans of data, filesystem metadata, and locations of files within spans. In SOCI, zTOCs are many-to-one with image layers, that is, there may be many different zTOCs for a single image layer.
+
+zTOCs are used by the SOCI snapshotter to extract individual files from an image layer using the following algorithm:
+1) Use the file locations from the zTOC to determine which spans contain the file
+2) Use the span metadata to determine where these spans exist in the compressed layer
+3) Use the compression context from the span metadata as well as the information from step 2 to decompress those spans out of the compressed layer
+4) Use the file locations to get the subset of bytes within the spans that correspond to the file
+5) Combine the file data with the filesystem metadata to reconstruct the full file
+
+__span__: A chunk of data that can be independently decompressed. A zTOC contains periodic checkpoints of compression state from which a process can resume decompression. The chunk of data between two checkpoints is a span.
+
+
+## Anti-terminology
+
+__SOCI Image__: We generally avoid the term "SOCI Image" because within SOCI, the image is an unmodified OCI image. A single image may have many SOCI indices with different parameters such as span size, layers indexed, etc. The precise way to refer to an image that has a SOCI index is to refer to the index itself. 
```

Comment:
Suggest saying it more succinctly and whimsically:
> We generally avoid the term "SOCI Image" because there is no such thing! The image is an unmodified OCI image. Also, a single image may have...